cmd/aries-agentd/main.go

```diff
@@ -38,6 +40,7 @@ import (
 
 const agentHostEnvKey = "ARIESD_API_HOST"
 const agentHTTPInboundEnvKey = "ARIESD_INBOUND_HOST"
+const agentDBPathEnvKey = "ARIESD_DB_PATH"
```

Comment:
It will be nice when we actually handle this with cobra :).

Reply:
#94